### PR TITLE
Rename authority service to identity service

### PR DESCRIPTION
**Pull request by Danielius1922**

### CODE REVIEW

**Comment by jkralik** [comment]
please update this doc https://plgd.dev/tutorials/external-oauth-server/

File: bundle/README.md [hunk 1]
```diff
@@ -31,47 +36,48 @@ Image can be configured via environment variables as argument `-e ENV=VALUE` of
...
+### Device Onboarding
+
 The onboarding values which should be set to the [coapcloudconf](https://github.com/openconnectivityfoundation/cloud-services/blob/c2c/swagger2.0/oic.r.coapcloudconf.swagger.json) device resource are:
 
-### Unsecured device
+#### Unsecured device
+
 | Attribute | Value |
 | --------- | ------|
 | `apn` | `test` |
```
Comment (jkralik) [inline bundle/README.md]: `plgd`

File: bundle/README.md [hunk 2]
```diff
@@ -1,25 +1,30 @@
...
+### Features
+
 - [OCF Native Cloud](https://openconnectivity.org/specs/OCF_Device_To_Cloud_Services_Specification_v2.1.0.pdf)
 - OAUTH Athorization code is not verified
 - [GRPC](https://github.com/plgd-dev/cloud/blob/master/grpc-gateway/pb/service.proto)
 
-## Supported clients
+### Supported clients
+
 - [iotivity v2+](https://github.com/iotivity/iotivity)
```
Comment (jkralik) [inline bundle/README.md]: `iotivity v2+` is not supported any more

File: bundle/README.md [hunk 3]
```diff
@@ -96,18 +101,19 @@ make CLOUD=1 SECURE=0 cloud_server cloud_client
...
 ./cloud_client cloud_client test coap+tcp://127.0.0.1:5683 00000000-0000-0000-0000-000000000001 plgd
-
 ```
 
-### Secured device
+#### Secured device
+
 | Attribute | Value |
 | --------- | ------|
 | `apn` | `test`|
```
Comment (jkralik) [inline bundle/README.md]: `plgd`

File: bundle/README.md [hunk 4]
```diff
@@ -151,11 +156,12 @@ docker exec -it cloud cat /data/certs/root_ca.crt > pki_certs/cloudca.pem
 ./onboarding_tool
 
 # Then restart device sample and client
-
 ```
 
-## Build a COAP client application
+### Build a COAP client application
+
 To build the client you need to have **golang v1.13+**.
```
Comment (jkralik) [inline bundle/README.md]: 1.16+

File: bundle/README.md [hunk 5]
```diff
@@ -164,8 +170,10 @@ go build
 ./coap --signUp test --href /oic/res
 ```
 
-## Build a GRPC client application
+### Build a GRPC client application
+
 To build the client you need to have **golang v1.13+**.
```
Comment (jkralik) [inline bundle/README.md]: 1.16+

File: bundle/run.sh [hunk 6]
```diff
@@ -596,7 +568,8 @@ cat /configs/coap-gateway.yaml | yq e "\
   .apis.coap.address = \"${COAP_GATEWAY_UNSECURE_ADDRESS}\" |
   .apis.coap.externalAddress = \"${FQDN}:${COAP_GATEWAY_UNSECURE_PORT}\" |
   .apis.coap.tls.enabled = false |
-  .apis.coap.authorization.providers[0].name = \"${DEVICE_PROVIDER}\" | 
+  .apis.coap.authorization.ownerClaim = \"https://${OWNER_CLAIM}\" |
```
Comment (jkralik) [inline bundle/run.sh]: just `.. = \"${OWNER_CLAIM}\" |`

File: identity/service/service.go [hunk 7]
```diff
@@ -4,31 +4,28 @@ import (
...
-	"github.com/plgd-dev/cloud/pkg/security/jwt"
 	"github.com/plgd-dev/cloud/pkg/security/jwt/validator"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/eventbus/nats/client"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/eventbus/nats/publisher"
 	"github.com/plgd-dev/cloud/resource-aggregate/cqrs/utils"
+	"google.golang.org/grpc"
 )
 
-// Provider defines interface for authentification against auth service
+// Provider defines interface for authentication against auth service
```
Comment (jkralik) [inline identity/service/service.go]: pls remove the comment - it doesn't below to `Persistance`

File: resource-directory/test/service.go [hunk 8]
```diff
@@ -18,11 +18,9 @@ func MakeConfig(t *testing.T) service.Config {
 	cfg.APIs.GRPC.Config = config.MakeGrpcServerConfig(config.RESOURCE_DIRECTORY_HOST)
 	cfg.APIs.GRPC.OwnerCacheExpiration = time.Minute
 
-	cfg.Clients.AuthServer.CacheExpiration = time.Millisecond * 50
-	cfg.Clients.AuthServer.PullFrequency = time.Millisecond * 200
-	cfg.Clients.AuthServer.OwnerClaim = config.OWNER_CLAIM
-	cfg.Clients.AuthServer.Connection = config.MakeGrpcClientConfig(config.AUTH_HOST)
-	cfg.Clients.AuthServer.OAuth = config.MakeOAuthConfig()
+	cfg.Clients.IdentityServer.CacheExpiration = time.Millisecond * 50
+	cfg.Clients.IdentityServer.PullFrequency = time.Millisecond * 200
```
Comment (jkralik) [inline resource-directory/test/service.go]: I think these 2 variables could be removed too. It was used with `userDevicesCache` which was replaced by `ownerCache`

File: identity/service/service.go (hunk 7 above)
Comment (Danielius1922) [reply identity/service/service.go]: Removed

File: resource-directory/test/service.go (hunk 8 above)
Comment (Danielius1922) [reply resource-directory/test/service.go]: Indeed they are no longer needed, so they have been removed.

File: bundle/run.sh (hunk 6 above)
Comment (Danielius1922) [reply bundle/run.sh]: Fixed

File: bundle/README.md (hunk 4 above)
Comment (Danielius1922) [reply bundle/README.md]: Fixed

File: bundle/README.md (hunk 2 above)
Comment (Danielius1922) [reply bundle/README.md]: Removed

File: bundle/README.md (hunk 1 above)
Comment (Danielius1922) [reply bundle/README.md]: Fixed

**Review by jkralik — APPROVED** [review]
Greate job 👍

**Pull request merged**